lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}

```

Comment:
Thinking for 5 more seconds.. That means we'd have to put the WebSocket URL in every subscription file, which makes updating it a bit of a chore.